Packaging/OpenDingux/package-opk.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+readonly OUT="${1:-../../build/devilutionx.opk}"
+readonly IN="${2:-../../build/devilutionx.dge}"
+
+readonly TMP="tmp/opk"
+
+echo 1>&2 Packaging ${OUT}...
+
+set -x
+rm -rf "${TMP}"
+mkdir -p "${TMP}"
+cat > "${TMP}/default.gcw0.desktop" <<EOF
```

Comment:
We should probably think about making a general .desktop template for all unix systems

Reply:
This is rather specific to Dingux. There is a custom manual and a custom flag to enable the controller.

Reply:
Maybe they could be added to the general one, not saying we have to do this now, but maybe good to keep in mind.

Reply:
There is also a custom launch command to add SDL2 controller mappings,
so it's not just additive:

https://github.com/glebm/devilutionX/blob/7ad53fff896433e56cbcd18858f8bda8395adb72/Packaging/OpenDingux/package-opk.sh#L20-L30

But yeah maybe a template in the future